README.md

```diff
@@ -103,12 +45,19 @@ To update dependencies, run
 
     dep ensure --update && dep prune
 
+## Further reading
+
+To learn more about Cortex, consult the following documents / talks:
+
+- [Original design document for Project Frankenstein](http://goo.gl/prdUYV)
+- PromCon 2016 Talk: "Project Frankenstein: Multitenant, Scale-Out Prometheus": [video](https://youtu.be/3Tb4Wc0kfCM), [slides](http://www.slideshare.net/weaveworks/project-frankenstein-a-multitenant-horizontally-scalable-prometheus-as-a-service)
```

Comment:
I'd like to keep it. This is a GitHub repo and we can presume the audience is technical. The presence of a design doc is a good signal for many, even if they don't read it.